spec/dummy/spec/rails_helper.rb

```diff
@@ -29,7 +29,7 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
```

Comment:
I agree. My heart ached when I did this. But the new test I added for formatter didn't go well with the in built way of test data cleaning. It kept saying no active transactions to rollback. Which was kind of the right error. Hence I ended up switching to data cleaner which handled it better.